scripts/test.js

```diff
@@ -1,13 +1,24 @@
-//@ts-check
-const cp = require("child_process");
-const path = require("path");
-const fs = require("fs");
-const os = require("os");
-const { rescript_exe } = require("#cli/bin_path");
-
-const duneBinDir = require("./dune").duneBinDir;
-
-const { exec } = require("../tests/build_tests/utils.js");
+// @ts-check
+
+import * as fs from "node:fs";
+import * as os from "node:os";
+import * as path from "node:path";
+import {
+  buildTestDir,
+  compilerTestDir,
+  docstringTestDir,
+  ounitTestBin,
+  projectDir,
+} from "#dev/paths";
```

Comment:
Never seen this `#dev/` syntax.
This is configured by `"imports"` in package.json?

Reply:
Yep. It's Node.js feature since v14, supported by TypeScript `Node16` module resolution.

https://nodejs.org/api/packages.html#subpath-imports